cypress/integration/calculator-homework-Indre-Kelly/calculator.spec.js

```diff
@@ -0,0 +1,66 @@
+import Calculator from '../../page_objects/calculator_main/Calculator.js';
+
+var calculator = new Calculator;
+const builds = ["Prototype", "1", "2", "3", "4", "5", "6", "7", "8", "9"]
+
+describe("Test Suit", () => {
+    beforeEach("Visit BasicCalculator", () => {
+        cy.visit("https://testsheepnz.github.io/BasicCalculator")
+    })
+        for (let i = 0; i < builds.length; i++) {
+            it.only("Can add numbers", () => {
+                cy.get("#selectBuild").select(builds[i]);
+                calculator.getFirstNumberField().type("8");
+                calculator.getSecondNumberField().type("2");
+                calculator.getOperationDropdown().select("Add");
+                calculator.getCalculateButton().click();
+                calculator.getAnswerField().should("have.value", "10");
+                calculator.getIntegerCheckBox().click();
```

Comment:
You click integer checkbox, but do not check the answer. Is this not needed? (same for other operations)